test/integration/src/py/test_liquidity_pools.py

```diff
@@ -282,6 +393,127 @@ def test_request_faucet_coins(
+    get_faucet_balance(basic_transfer_request.sifnodecli_node)
+    balance = test_utilities.get_sifchain_addr_balance(sifaddress, basic_transfer_request.sifnodecli_node, "rowan")
+     
+    
+    #ceth
+    start_faucet_ceth_ballance = get_currency_faucet_balance("ceth", get_faucet_balance(basic_transfer_request.sifnodecli_node))
+    get_faucet_balance(basic_transfer_request.sifnodecli_node)
+    basic_transfer_request.amount = 10
+    basic_transfer_request.sifchain_symbol = "ceth"
+    basic_transfer_request.sifchain_address = sifaddress
+    # add
+    add_faucet_coins(basic_transfer_request, credentials)
+    time.sleep(20)
+    after_add_faucet_ceth_ballance = get_currency_faucet_balance("ceth", get_faucet_balance(basic_transfer_request.sifnodecli_node))
+    assert (start_faucet_ceth_ballance < after_add_faucet_ceth_ballance), "faucet ballance in ceth should be higher after adding coins to faucet"
+    get_faucet_balance(request.sifnodecli_node)
+    # request
+    basic_transfer_request.amount = 5
+    request_faucet_coins(basic_transfer_request, credentials)
+    time.sleep(20)
+    get_faucet_balance(basic_transfer_request.sifnodecli_node)
+    balance_ceth = test_utilities.get_sifchain_addr_balance(sifaddress, basic_transfer_request.sifnodecli_node, "ceth")
+    
+    #random currency
+    start_faucet_random_ballance = get_currency_faucet_balance(sifchain_symbol, get_faucet_balance(request.sifnodecli_node))
+    get_faucet_balance(request.sifnodecli_node)
+    request.amount = 20000
+    request.sifchain_symbol = sifchain_symbol
+    request.sifchain_address = sifaddress
+    # add
+    add_faucet_coins(request, credentials)
+    time.sleep(20)
+    after_add_faucet_random_ballance = get_currency_faucet_balance(sifchain_symbol, get_faucet_balance(request.sifnodecli_node))
+    assert (start_faucet_random_ballance < after_add_faucet_random_ballance), "faucet ballance in random currency should be higher after adding coins to faucet"
+   
+    get_faucet_balance(request.sifnodecli_node)
+    balance_new_currency = test_utilities.get_sifchain_addr_balance(sifaddress, request.sifnodecli_node, sifchain_symbol)
+    logging.info(sifchain_symbol+" balance after add, should be lower="+ str(balance_new_currency))
+    logging.info(get_faucet_balance(request.sifnodecli_node))
+    # request
+    request.amount = 1000
+    request_faucet_coins(request, credentials)
+    time.sleep(20)
+    get_faucet_balance(request.sifnodecli_node)
+    logging.info(get_faucet_balance(request.sifnodecli_node))
+    balance_new_currency = test_utilities.get_sifchain_addr_balance(sifaddress, request.sifnodecli_node, sifchain_symbol)
+    
+    #get faucet balances
+    logging.info(sifchain_symbol+" balance after request should be back="+ str(balance_new_currency))
+    logging.info("ceth  balance="+ str(balance_ceth))
```

Comment:
Using formatted logging is stylistically nicer.  Like:

`logging.info("ceth balance={balance_ceth}")`

Reply:
done